lib/active_shipping/location.rb

```diff
@@ -13,6 +13,7 @@ class Location
       address3: [:address3],
       phone: [:phone, :phone_number],
       fax: [:fax, :fax_number],
+      email: [:email, :email_address],
```

Comment:
Thought it would be useful in the library, but I think it's okay to drop it in this case.